tests/e2e/pageobjects/ide/ProjectTree.ts

```diff
@@ -250,20 +253,22 @@ export class ProjectTree {
             await this.waitItemExpanded(rootItem);
 
             // do five checks of the item in one fifth of the time given for root folder item (was causing frequent reloads of the workspace)
-            const isRootSubItemVisible = await this.driverHelper.waitVisibilityBoolean(rootSubitemLocator, 5, visibilityItemPolling / 5);
+            const isSubfolderVisible = await this.driverHelper.waitVisibilityBoolean(rootSubitemLocator, 5, visibilityItemPolling / 5);
 
-            if (!isRootSubItemVisible) {
+            if (!isSubfolderVisible) {
                 Logger.trace(`ProjectTree.waitProjectImported sub-items not found, reloading page.`);
                 await this.browserTabsUtil.refreshPage();
                 await this.ide.waitWorkspaceAndIde();
                 await this.openProjectTreeContainer();
+                if (i === attempts - 1) {
+                    throw new error.TimeoutError('Exceeded the maximum number of checking attempts, project has not been imported [unable to locate project subfolder]');
+                }
                 continue;
             }
-            return;
-        }
-
-        throw new error.TimeoutError('Exceeded the maximum number of checking attempts, project has not been imported');
 
+            Logger.trace(`ProjectTree.waitProjectImported project successfully imported`);
+            break;
```

Comment:
`break` looks redundant in here

Reply:
it has a purpose, it prevents false negatives, stops the for loop.
I could replace it with `return;` if necessary